framgia/sph_ui-ux-swipet
# [Task] Eslint Setup

## Related Links

- Task Link: https://app.asana.com/0/1202252431396579/1202264718826109/f
- Route Link: /

## Description

- Setup Eslint

## Notes
**How to Test**

1. Run `npm install`
2. Disable `eslint` and `prettier` VScode extensions while testing _(if ever you install these)_ 
3. Try to replace single quote to double qoutes `className=""` 
4. It should prompt an error in the terminal
<img width="723" alt="image" src="https://user-images.githubusercontent.com/70557829/169245766-edaee27d-b1bd-4034-950f-0fa0fe50bca5.png">



## Screenshots

N/A
